lib/mapping.ts

```diff
@@ -0,0 +1,302 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+'use strict'
```

Comment:
you don't need this

Reply:
Just because It was in the previous code-base. We will remove it.